src/PullRequestReleaseNotes/Providers/GitHubPullRequestProvider.cs

```diff
@@ -40,28 +40,28 @@ public PullRequestDto Get(string commitMessage)
             var pullRequestId = ExtractPullRequestNumber(commitMessage);
             if (pullRequestId == null)
                 return null;
-            var pullRequest = _gitHubClient.PullRequest.Get(_programArgs.GitHubOwner, _programArgs.GitHubRepository, (int) pullRequestId).Result;
-            if (pullRequest == null)
-                return null;
-            if (_programArgs.VerboseOutput)
-                Console.WriteLine($"Found #{pullRequest.Number}: {pullRequest.Title}");
-            return GetPullRequestWithLabels(pullRequest);
+            return GetPullRequestWithLabels(pullRequestId.Value);
```

Comment:
this library has the id, which is a bit confusing: https://github.com/eclipse/egit-github/blob/master/org.eclipse.egit.github.core/src/org/eclipse/egit/github/core/PullRequest.java

```java
public class PullRequest implements Serializable {
        private long id;
        ......
        private int number;
```